requirements.in

```diff
@@ -1,13 +1,15 @@
 click
 docker-py
-PyYAML
-requests
+#PyYAML
```

Comment:
I guess these should be uncommented?

Reply:
The same file has `requests>=2.20.0` and `pyyaml>=4.2b1` below which may lead to double requirements issues. :thinking: